src/applications/static-pages/representative-status/utilities/phoneNumbers.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @typedef {Object} ParsedPhoneNumber
+ * @property {string} contact - the "raw" phone number, used to populate the tel: link
+ * @property {string} extension - the extension, can be any number of digits
+ */
+
+/**
+ * Parses a phone number string for use with the Telephone component.
+ * @param phone {String} "raw" phone number. The first 10 digits are treated as the "main" number.
+ * Any remaining digits are treated as the extension.
+ * Non-digits preceding the extension are ignored.
+ * @returns {ParsedPhoneNumber}
+ */
+
+export const parsePhoneNumber = phone => {
+  if (!phone) {
+    return { contact: null, extension: null };
+  }
+  let sanitizedNumber = phone
+    .replace(/[()\s]/g, '') // remove parentheses
+    .replace(/(?<=.)([+.*])/g, '-'); // replace .*+ symbols being used as dashes
+
+  // return null for non-US country codes
+  if (sanitizedNumber.match(/\+(\d+)[^\d1]/g)) {
```

Comment:
if `.replace(/(?<=.)([+.*])/g, '-'); // replace .*+ symbols being used as dashes` works, then how would this condition looking for `+` be true?

Reply:
The way I'm determining whether a `+` is "being used as a dash" is if it follows another character. So a number formatted like `+1+555+123+4567` would be converted to `+1-555-123-4567`.

Reply:
I missed that, thanks!